janus_core/helpers/observables.py

```diff
@@ -0,0 +1,75 @@
+"""
+Module for built-in correlation observables.
+"""
+
+from ase import Atoms, units
+
+
+# pylint: disable=too-few-public-methods
+class Stress:
+    """
+    Observable for stress components.
+
+    Parameters
+    ----------
+    component : str
+        Symbol for tensor components, xx, yy, etc.
+    include_ideal_gas : bool
+        Calculate with the ideal gas contribution.
+    """
+
+    def __init__(self, component: str, include_ideal_gas: bool = True):
```

Comment:
Very specific setting, recommend making that kwarg only
```suggestion
    def __init__(self, component: str, *, include_ideal_gas: bool = True):
```

Reply:
Yes I think that is better thanks